src/material/autocomplete/autocomplete.ts

```diff
@@ -291,6 +293,13 @@ export abstract class _MatAutocompleteBase
     return this.ariaLabelledby ? labelExpression + this.ariaLabelledby : labelId;
   }
 
+  /** Returns the theme to be used on the panel. */
+  _getPanelTheme() {
+    return this._parentFormField ? `mat-${this._parentFormField.color}` : '';
+  }
+
+  protected _parentFormField: MatFormField | null = inject(MAT_FORM_FIELD, {optional: true});
```

Comment:
I don't think that we should be injecting the form field here since the panel isn't guaranteed to be inside of a form field and it can be attached to multiple triggers. Instead we should be taking the current `color` in the `MatAutocompleteTrigger` and forwarding it to the panel one open. See https://github.com/angular/components/blob/main/src/material/autocomplete/autocomplete-trigger.ts#L668

Reply:
good idea, I changed it to use the trigger